vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -296,6 +296,12 @@ export default class SbcHeader extends NavigationMixin {
       window.location.assign(`${ConfigHelper.getAuthContextPath()}home`)
     }
   }
+
+  getContextPath (): string {
```

Comment:
Does this assume that the client application will have a signIn and signOut view like auth?

Reply:
Yes, they should have a signin and signout view, which we are already mentioned in the readme doc